eth/protocol.go

```diff
@@ -108,15 +106,24 @@ type txPool interface {
 	SubscribeNewTxsEvent(chan<- core.NewTxsEvent) event.Subscription
 }
 
-// statusData is the network packet for the status message.
-type statusData struct {
+// statusDataLegacy is the network packet for the status message for eth/63.
+type statusDataLegacy struct {
```

Comment:
Why not use `statusDataV63` and `statusDataV64` ?

Reply:
Not sure, wanted to signal clearer that we should get rid of it.

Reply:
I've renamed the legacy one it to `statusData63`. I don't want to add 64 to the new one yet since I don't think 65/66 will modify it, so let's keep it unversioned until there's a modification. Then we can tag it.